user-guide/Standard_Apps/EPM/EPM_I-DOCSIS/I-DOCSIS_deployment.md

```diff
@@ -38,36 +43,54 @@ To deploy the I-DOCSIS branch of the EPM Solution:
-     > - The DataMiner element names must match the CCAP names as defined in SNMP. This is required to properly link the topology visuals to the elements. To rename a CCAP Platform element, first change the user-defined CCAP name in SNMP to the new name, then rename the element in Surveyor to match the new name.
+   > [!NOTE]
+   > We recommend that you avoid placing collector or back-end elements on the same DMA as the front-end element.
 
-   - **Back-end EPM elements**, in the *System* > *DataMiner EPM* > *EPM BE* view.
-   - A **front-end EPM element**, in the *System* > *DataMiner EPM* > *EPM FE* view.
-   - **System elements**, as necessary in the *System* view. You can name these elements as you see fit. If you add any MS/LX Platform elements, add these in a *DMS* subview of the *System* view.
+1. Assign the necessary Visio drawings to the EPM elements:
 
-1. In the Automation module in DataMiner Cube, **import the Automation scripts** from the EPM package. See [Importing and exporting Automation scripts](xref:Managing_Automation_scripts#importing-and-exporting-automation-scripts).
+   1. Go to *Apps* > *Protocols & Templates*.
 
-1. If the EPM package contains any **dashboards**, add these to the DMS. To do so, copy the dashboards to the folder `C:\Skyline DataMiner\Dashboards` on the target DMA.
+   1. Select *Skyline EPM Platform*.
 
-1. If the EPM package contains Visio drawings (in the *Visuals* folder), **add the Visio drawings** to the DMS. To do so:
+   1. Under *Visio Files*, right-click *Custom* and select *Set as active Visio file*.
 
-   - For a Visio drawing linked to a specific connector (e.g. EPM Platform), see [Assigning a custom Visio file to a protocol](xref:Managing_Visio_files_linked_to_protocols#assigning-a-custom-visio-file-to-a-protocol).
-   - For a Visio drawings linked to a view, upload the drawing to the view. See [Set as active Visio file](xref:Editing_a_visual_overview_in_DataMiner_Cube#set-as-active-visio-file).
+   1. Select *Skyline EPM Platform DOCSIS* and repeat the previous step.
 
-1. **Add the Correlation rules** from the EPM package to the DMS. To do so, copy the Correlation rules to the folder `C:\Skyline DataMiner\Correlation` of the target DMA and **restart** the DMA.
+1. Configure the front-end element:
 
-   > [!NOTE]
-   > As these Correlation rules require specific connectors and Automation scripts to function properly, make sure these have all been loaded properly before you add the Correlation rules. If you are deploying the EPM Solution for the first time, the necessary elements need to be created first before the Correlation rules can be deployed. We also recommend that you review the Correlation rules both when you initially deploy them and during each upgrade.
+   1. Add the DMA ID/Element ID of the front-end element to the *Frontend Registration* table. This can be found in the Element in the Configuration Topology Chain under the Frontend page.
```

Comment:
@SkylineGBO, this isn't entirely clear. Do you mean "in the Configuration Topology chain on the "Frontend" page of the front-end element"?